### PR TITLE
Newmaster

### PR DESCRIPTION
**Pull request by alisman**

### CODE REVIEW

File: src/lib/CSVExport.ts [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+export function CSVGenerate(requests) {
+  const csvString = [
+    StaticTableColumns.map(item => item.label),
+    requests.map(req => StaticTableColumns.map(col => req[col.dataKey!]))
+  ]
+
+    .map(e => e.join("\t"))
+    .join("\n");
+
```
Comment (ao508) [inline src/lib/CSVExport.ts:12]: pending some updates from shalabh to get the export working as desired

additionally, would you mind adding a prompt for the user for where to save the file and name it?

Comment (ao508) [inline src/lib/CSVExport.ts:12]: pending some updates from shalabh to get the export working as desired

additionally, would you mind adding a prompt for the user for where to save the file and name it?

File: src/pages/requestView/RequestSummary.tsx [hunk 2]
```diff
@@ -1,146 +1,153 @@
...
+  //         offset: startIndex,
+  //         limit: stopIndex
+  //       }
+  //     }
+  //   });
+  // }
+
+  // function isRowLoaded({ index }) {
+  //   return index < data.requests[-].length;
+  // }
```
Comment (ao508) [inline src/pages/requestView/RequestSummary.tsx:45]: is this commented out for later?

if we're moving away from needing this then we can just remove it

File: src/shared/components/SmileNavBar.tsx [hunk 3]
```diff
@@ -0,0 +1,271 @@
...
+        </a>
+        {/*<i className="bi bi-list toggle-sidebar-btn"></i>*/}
+      </div>
+
+      {/*<div className="search-bar">*/}
+      {/*  <form className="search-form d-flex align-items-center" method="POST" action="#">*/}
+      {/*    <input type="text" name="query" placeholder="Search" title="Enter search keyword" />*/}
+      {/*      <button type="submit" title="Search"><i className="bi bi-search"></i></button>*/}
+      {/*  </form>*/}
+      {/*</div>*/}
```
Comment (ao508) [inline src/shared/components/SmileNavBar.tsx:21]: i think we can remove this also

**Review by ao508 — CHANGES_REQUESTED** [review]

File: src/lib/CSVExport.ts (hunk 1 above)
Comment (alisman) [reply src/lib/CSVExport.ts:12]: We can add a custom prompt for the file name but the save location is feature of the browser that we can't configure/force.  It just depends how user has it set up (e.g. always just download to Downloads folder)

File: .prettierignore [hunk 4]
```diff
@@ -30,3 +30,6 @@
 LICENSE
 packages/**/dist/**
 yarn.lock
+
```
Comment (ao508) [inline .prettierignore:33]: based on the output from `prettier` I think we'll need to add `*.graphql` to this file

**Review by ao508 — CHANGES_REQUESTED** [review]
Need to check in the modified files that `prettier` generates for the following (based on the circleci test output):

- src/generated/graphql.tsx
- src/index.css
- src/pages/requestView/RequestSummary.tsx

**Pull request closed without merging**